app/models/concerns/state_behavior.rb

```diff
@@ -2,38 +2,11 @@ module StateBehavior
   extend ActiveSupport::Concern
 
   included do
-    before_update :check_state
-
-    validates_with StateValidator
-
-    def check_state
-      return unless state_changed?
-      complete_record if state == 'complete'
-      ReviewerMailer.notify(id, state).deliver_later
+    # Sipity workflow state name
+    def workflow_state
+      to_sipity_entity.workflow_state.name
+    rescue RuntimeError
```

Comment:
Right. This is mainly about catching issues that arise during testing. Not all tests require full sipity workflows to be created, but this method gets called and throws an error.